src/app/pages/account/users/user-form/user-form.component.ts

```diff
@@ -202,24 +207,24 @@ export class UserFormComponent {
           type : 'checkbox',
           name : 'locked',
           placeholder : T('Lock User'),
-          tooltip : T('Disables logging in to this user account until\
-            this option is unchecked.'),
+          tooltip : T('Disable logging in to this user account.'),
           isHidden: false
         },
         {
           type : 'checkbox',
           name : 'sudo',
           placeholder : T('Permit Sudo'),
-          tooltip : T('Give this user permission to use\
-            <a href="https://www.sudo.ws/" target="_blank">sudo</a>.'),
+          tooltip : T('Give this user permission to use <a '
+                    + 'href="https://www.sudo.ws/" '
+                    + 'target="_blank">sudo</a>.'),
           isHidden: false
         },
         {
           type : 'checkbox',
           name : 'microsoft_account',
           placeholder : T('Microsoft Account'),
-          tooltip : T('Check if the user will be connecting from a\
-            Windows 8 or higher system.'),
+          tooltip : T('Check when the user connects from a '
```

Comment:
After consultation with John:
'Set to allow additional username authentication methods when the user is connecting from a Windows 8 or newer operating system.'